docs/docs/cmd/onenote/notebook/notebook-add.md

```diff
@@ -0,0 +1,127 @@
+# onenote notebook add
+
+Create a new OneNote notebook.
+
+## Usage
+
+```sh
+m365 onenote notebook add [options]
+```
+
+## Options
+
+`-n, --name <name>`
+: Name of the notebook. Notebook names must be unique. The name cannot contain more than 128 characters or contain the following characters: ?\*/:<>
+
+`--userId [userId]`
+: Id of the user. Use either userId or userName, but not both.
+
+`--userName [userName]`
+: Name of the user. Use either userId or userName, but not both.
+
+`--groupId [groupId]`
+: Id of the SharePoint group. Use either groupName or groupId, but not both
+
+`--groupName [groupName]`
+: Name of the SharePoint group. Use either groupName or groupId, but not both.
+
+`-u, --webUrl [webUrl]`
+: URL of the SharePoint site.
```

Comment:
seems to me the notebook may be created either for user (by name or id) or for group (by name or id) or for a given site right? It is not possible to specify user and group at the same time, correct? If so than I thing we should align the description to point it out. Currently it would seem I may do something like 
```
m365 onenote notebook add --name "Private Notebook" --userName user1@contoso.onmicrosoft.com --groupName "MyGroup"
```